extensions/machine-learning-services/src/common/processService.ts

```diff
@@ -11,6 +11,16 @@ import * as childProcess from 'child_process';
 const ExecScriptsTimeoutInSeconds = 600000;
 export class ProcessService {
 
+	private _timeout = ExecScriptsTimeoutInSeconds;
+
+	public set Timeout(value: number) {
+		this._timeout = value;
+	}
+
+	public get Timeout(): number {
```

Comment:
If these are both public why even bother having the property definitions specifically?